### PR TITLE
Makes Jones mechscannable with a syndicate scanner

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
Jones can be mechscanned with a syndicate scanner, Jones costs 2.5 viscerite to print out.

## Changelog 

```changelog
(u)Ikea
(+)Jones can now be mechscanned with a syndicate analyzer
```

### CODE REVIEW

**Comment by CheffieGithub** [comment]
This allowing for mob critter scanning is important and probably should be mentioned in the PR or changelog. Will help greatly for mobification of robotic critters in future.

**Comment by ThatFiveGuys** [comment]
> This allowing for mob critter scanning is important and probably should be mentioned in the PR or changelog. Will help greatly for mobification of robotic critters in future.

With your comment its mentioned in the pr now

**Comment by CheffieGithub** [comment]
it also costs 2.5 viscerite since 1 material is 0.1

**Comment by nefarious6th** [comment]
Speedmerge

**Comment by pali6** [comment]
Catmandu gives a precedent for Jones clones so it weirdly fits

File: _std/macros/antag_popups.dm [hunk 1]
```diff
@@ -335,6 +335,10 @@
 	if (src.holder)
 		get_singleton(/datum/antagPopups).showPanel()
 
+
+TYPEINFO(/mob)
+	var/list/mats = 0
```
Comment (pali6) [inline _std/macros/antag_popups.dm:340]: Why not just put it on atom/movable then?

Comment (ThatFiveGuys) [reply _std/macros/antag_popups.dm:340]: atom/movable includes things like screens, would moving it to atom/movable have a performance impact (even if a minor one)?

Comment (ThatFiveGuys) [reply _std/macros/antag_popups.dm]: Moved to atom/movable

Comment (pali6) [reply _std/macros/antag_popups.dm]: no

Comment (ThatFiveGuys) [reply _std/macros/antag_popups.dm]: No as in doesnt have a performance impact, not no as in I didnt move it to atom/movable right?

**Review by pali6 — APPROVED** [review]
🐈🔧

**Pull request merged**